src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/form.html.twig

```diff
@@ -59,6 +59,10 @@
           </div>
         </div>
 
+        {{ ps.form_group_row(employeeForm.has_enabled_gravatar, {}, {
+          'label': 'Use gravatar'|trans({}, 'Admin.Global')
```

Comment:
What terminology best suits the field: 'Enable Gravatar', 'Connect to Gravatar' or 'Use Gravatar'?

Reply:
I think `Enable Gravatar` is the best terminology.

`Connect to Gravatar` is not the case, because there is not auth.
`Use Gravatar` could be a second terminology, but I think Enable is better

Reply:
```suggestion
          'label': 'Enable Gravatar'|trans({}, 'Admin.Global')
```

Reply:
@LouiseBonnard Wording applied